app.js

```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const app = express();
+const { getTopics } = require('./controller/controller')
+
+app.use(express.json());
+
+app.get('/api/topics', getTopics);
+
+// Error Handling
+app.use((err, req, res, next) => {
+    console.error(err);
+  
+    if (err.code) {
+      switch (err.code) {
+        case '23502':
+        case '22P02':
+        case '23503':
```

Comment:
have you come across 23503 yet? I would have thought you would only see this on a future ticket